data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip. She opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" the Alpha shouts, stopping his attempts to break free.`
+				goto detained
+			label notnavy
+			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
+			label detained
+			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
+			`	After the short verdict, the agents bring the Alpha to a lower level of the building, with several empty cells kept shut by thick, tough walls of some transparent material. Only a few small holes are available for airflow. They lock him in one of the cells, making him the only prisoner in the largely empty building. The agents then ask that you wait a few minutes for a medic to arrive, so that they can take care of your bruises.`
+			`	"Don't know why they bother with that, it's not like I hit you too hard," the Alpha says. "I guess I owe you an apology, though. I've given up on fighting humanity left and right, really, I just got a bit jumpy when I spotted you."`
+			choice
+				`	"Who are you, exactly? How did you even get here?"`
+				`	"What are you doing here in the Coalition?"`
+			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s the crew I was on found some aliens somewhere far north of Earth. We didn't get to get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting, as the battle wouldn't be favorable to us, but we managed to loot a jump drive from one of the aliens. After that we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
```

Comment:
```suggestion
			`	"Where do I begin? Well, for starters, name's Alitis. Was Athis before, but I took a Saryd name after I got here. Back in the 2800s, the crew I was on found some aliens somewhere far north of Earth. We didn't get much from them though, as they were fighting another group of aliens, ones who had jump drives. We didn't want any part of the fighting because the battle wasn't favorable to us, but we managed to loot a jump drive from one of the aliens. After that, we started exploring disconnected systems to the east. Eventually, we joined an enclave in the Dirt Belt, but it didn't take long for the Navy to find us there. While evacuating, some of the others wanted to head back east, but I knew better, so I banked on heading further west, and after some time got here."`
```